index.html

```diff
@@ -0,0 +1,141 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Instruments</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <nav id="navbar">
+      <header>Music</header>
+      <a href="#main_types_of_instruments" class="nav-link" id="main-types">Main Types of Instruments</a>
```

Comment:
- [ ] Why don't you use a heading element or `<h1>` tag for the title instead of `<header>`, this will make the structure of the page more accurate.

The `<header>` element represents introductory content, it is typically used in the introduction or navigational aids, whereas the `<h1>` to `<h6>` tags are used for titles.

- [ ] Currently, all of your titles are using the `<header>` tag instead of the `<h1>` to `<h6>` tags, I would kindly advise you to change them, almost there, just a little bit more 🙌🏼